Bugzilla/Template.pm

```diff
@@ -260,6 +264,132 @@ sub quoteUrls {
     return $text;
 }
 
+sub quoteUrls2 {
```

Comment:
Most of this is left the same as the original `quoteUrls`. Just mainly change the bug link/attachment link functions to the new ones and commented out the autolinking code.